USER/civSettings.hpp

```diff
@@ -3,23 +3,21 @@
 *   In der Klasse "userEquipment" können eigene Ausrüstungsgegenstände für die Zivilisten definiert werden.
 */
 
+enableCivs = 0;
```

Comment:
gives us *one* switch for the whole module instead of enableOnFoot / enableInVehicles